### ISSUE TITLE
Server Side Example getting error in Resize Listener

### ISSUE THREAD

**Issue by mdvandemore** [EdCharbeneau/BlazorSize#15]
The code:

` private async ValueTask Cancel() =>
            await jsRuntime.InvokeVoidAsync($"{ns}.cancelListener");`

In file **ResizeListener.cs** is throwing **System.NullReferenceException** error when running the **BlazorSize.Example** project.  (It works fine with BlazorSize.CsbExample).

**Comment by EdCharbeneau** [comment]
I couldn't reproduce this. Can you provide more details?

**Comment by mdvandemore** [comment]
Thanks for checking this.   I'm running VS2019 16.5.3 / .Net 4.8.03752

Since I couldn't resolve the error, I created a generic Blazor Server-Side App, added BlazorSize from NuGet and the components from you test project and it works fine, so maybe I've got a weird debug setting?

If you want to try mine I've posted it here: https://github.com/mdvandemore/BlazorSizeTest

Thanks again, and thanks for creating BlazorSize.
